.eslintrc.json

```diff
@@ -36,7 +36,9 @@
       "files": [
         "src/utils/shared/*.?(js|ts)",
         "src/gatsby/**/*.js",
-        "src/components/PageWrapper/index.js",
+        "src/components/**/*.js",
```

Comment:
I just added the folders that includes `.js` files which is what dvc.org does, but is there a reason we can't just add `src/**/*.js` to this array?

Reply:
I'd prefer the catch-all `**` too...

Reply:
`**` should be fine, if we really need to ignore something we can add it to the ignore list anyway.